Cargo.toml

```diff
@@ -118,6 +119,7 @@ object = "0.32"
 rouille = { version = "3.6", optional = true, default-features = false, features = [
   "ssl",
 ] }
+shlex = "=1.3.0"
```

Comment:
please document why "=1.3.0"

Reply:
I think this is just what cargo selected when I did `cargo add shlex`. How do you suggest I update it?